### ISSUE TITLE
Not working anymore

### ISSUE THREAD

**Issue by red9350** [blastdan/Wox.Plugin.Choco#6]
It has stopped working. No results are fetched. See attached screenshot
![image](https://user-images.githubusercontent.com/16651290/121787019-ed95a280-cbc3-11eb-8e93-4683dac6a184.png)

**Comment by mangost** [comment]
True.

Not to waste your time unless it's updated.

**Comment by blastdan** [comment]
Yeah, the chocolatey API updates in a way that isn't supported occasionally.
I don't use the plugin anymore so it's hard to keep it updated.

**Comment by blastdan** [comment]
I pushed the update to the odata reference, but I don't remember the account details to publish it to the wox plugin library.

everything is so old. it needs to be redone.

**Comment by blastdan** [comment]
Alright, I have the new one updated.

If anyone can check to see if it even works that'd be nice.
